projects/valtimo/dossier/src/lib/dossier-list/dossier-list.component.ts

```diff
@@ -220,8 +239,14 @@ export class DossierListComponent implements OnInit {
     private readonly documentService: DocumentService,
     private readonly translateService: TranslateService,
     private readonly dossierService: DossierService,
-    private readonly logger: NGXLogger
-  ) {}
+    private readonly logger: NGXLogger,
+    private configService: ConfigService
+  ) {
+    this.dossierVisibleTabs = this.configService.config?.visibleDossierListTabs || null;
+    if (this.dossierVisibleTabs != null) {
```

Comment:
!==. Run lint on libs, I think you'll run into more errors, which will break pipeline if you don't fix